src/components/apps/Placeholder/Placeholder.svelte

```diff
@@ -14,15 +14,17 @@
 
     $motionVal = 1;
   });
+
+  $: imageTransform = !$prefersReducedMotion
+    ? `rotate(${180 * ($motionVal + 1)}deg) scale(${$motionVal}) translateZ(0px)`
+    : 'initial';
```

Comment:
Surely should be:

```js
  $: imageTransform = $prefersReducedMotion
    ? 'initial'
    : `rotate(${180 * ($motionVal + 1)}deg) scale(${$motionVal}) translateZ(0px)`;
```

Reply:
I kinda disagree. I like animated value to be specified first, even if it negates the ternary :)

Reply:
Oki